app/pulp/app/viewsets/repository.py

```diff
@@ -8,7 +8,7 @@
 from pulp.app.serializers import (ContentSerializer, ImporterSerializer, PublisherSerializer,
                                   RepositorySerializer, RepositoryGroupSerializer,
                                   RepositoryContentSerializer)
-from pulp.app.tasks import repository, tags
+from pulp.app.tasks import importer, repository, tags
```

Comment:
Another suggestion is to do `from pulp.app import tasks` and then `tasks.importer.delete.apply_async...` not to confuse task for importer and the instance of importer.

@mhrivnak , thanks for pointing to a simple solution here :)